docs/hub/rl-baselines3-zoo.md

```diff
@@ -6,6 +6,8 @@
 
 You can find RL-Baselines3-Zoo models by filtering at the left of the [models page](https://huggingface.co/models?library=stable-baselines3).
 
+And the documentation [here](https://stable-baselines3.readthedocs.io/en/master/guide/rl_zoo.html).
```

Comment:
Good point to link to the documentation! Perhaps we could do it in the introduction, so this section is only about the Hub?